include/oneapi/dpl/algorithm

```diff
@@ -59,21 +59,11 @@ using ::std::find;
 using ::std::find_if;
 using ::std::find_if_not;
 using ::std::for_each;
-using ::std::is_heap;
```

Comment:
Could you add information about this change in the description?

Reply:
Please read my mail.

Reply:
I see that it is fixed in #515. Let's remove these changes from this PR.